@ndlib/gatsby-theme-marble/gatsby-node.js

```diff
@@ -384,6 +384,7 @@ const defaultLayouts = {
                     { label: 'field', value: 'themeTag.keyword' },
                     { label: 'label', value: 'Keywords' },
                     { label: 'size', value: '10' },
+                    { label: 'sort', value: 'default' },
```

Comment:
What I would like to see is this value.  Set to a-z for the century facet.   Because that one expects to be in order by alphabetically.  Every other facet should get the default treatment.